workos/users.py

```diff
@@ -150,6 +150,18 @@ def list_users(
 
         return self.construct_from_response(response)
 
+    def delete_user(self, user):
+        """Delete an existing user.
+
+        Args:
+            user (str) -  User unique identifier
+        """
+        return self.request_helper.request(
```

Comment:
I know we do this in the other `delete` methods, but I think we might want to give this method a void return type (`None`, in Python) so that we're not leaking implementation details.